### PR TITLE
z v1.11

### PR DESCRIPTION
**Pull request by ericbn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/z.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Z < Formula
   desc "Tracks most-used directories to make cd smarter"
   homepage "https://github.com/rupa/z"
-  url "https://github.com/rupa/z/archive/v1.9.tar.gz"
-  sha256 "e2860e4f65770e02297ca4ca08ec1ee623a658bd9cc1acddbbe5ad22e1de70a7"
+  url "https://github.com/rupa/z.git",
```
Comment (fxcoudert) [inline Formula/z.rb]: We prefer to use tarballs

File: Formula/z.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
+    :tag => "v1.11",
```
Comment (fxcoudert) [inline Formula/z.rb]: 1.9 is still indicated as “latest release”

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/z.rb (hunk 2 above)
Comment (ericbn) [reply Formula/z.rb]: Opened https://github.com/rupa/z/issues/213 asking about this...

Comment (ilovezfs) [reply Formula/z.rb]: See https://github.com/rupa/z/issues/205.

**Comment by ilovezfs** [comment]
Thanks for the PR nonetheless!

**Pull request closed without merging**